src/components/Modals/Nfts/NftModal.tsx

```diff
@@ -110,6 +111,8 @@ export const NftModal: React.FC<NftModalProps> = ({ nftItem }) => {
     : null
   const customizeLink = nftCollection?.socialLinks?.find(link => link.name === 'customizeFoxatar')
 
+  const { refetch: refetchNft } = useGetNftQuery({ assetId: nftAssetId })
```

Comment:
Akschually, use the non-hook version and `initiate()` - this will fetch on component mount which we don't need

Alternatively, use https://redux-toolkit.js.org/rtk-query/api/created-api/hooks#uselazyquery - ty Phind for the find

Reply:
00e6c0481